packages/nimble-components/src/listbox/tests/listbox-matrix.stories.ts

```diff
@@ -0,0 +1,32 @@
+import { ViewTemplate, html } from '@microsoft/fast-element';
```

Comment:
Please add a "Hidden Listbox" story following the pattern of the other component matrix stories

Reply:
Sorry, I had excluded the "Hidden Listbox" story from my suggested change, because I didn't think we needed them for "internal" elements.

Reply:
I can see that argument if our internal usages never intend to hide the component. I lean towards adding it now because I suspect we may someday want to make this a public component or find an internal use case that wants to hide it and it's easier to remember to add the test coverage now.

Reply:
Added the `hidden listbox` test